habitat-hitl/habitat_hitl/_internal/hitl_driver.py

```diff
@@ -450,15 +449,6 @@ def sim_update(self, dt):
             + dt * self._hitl_config.viz_animation_speed
         ) % 1.0
 
-        # Navmesh visualization only works in the debug third-person view
```

Comment:
This kind of behavior tied to a keypress should be implemented in AppStates, not HitlDriver.